nUpdate.Shared/Updating/UpdateManager.cs

```diff
@@ -273,6 +275,60 @@ private void Cleanup()
             _packageOperations.Clear();
         }
 
+        /// <summary>
```

Comment:
Please remove the documentation here as this is a private method.

Reply:
Delete all documents Do not keep any comments?